src/helpers/getByAPI.js

```diff
@@ -95,13 +112,46 @@ const getNodeByTestId = (node, testID) => {
     : testID.test(node.props.testID);
 };
 
+export const nonErroringGetByText = (instance: ReactTestInstance) =>
+  function nonErroringGetByTextFn(text: string | RegExp) {
+    const matchingInstances = instance.findAll((node) =>
```

Comment:
Since we switch from `find` to `findAll`, the implementation doesn't throw anymore if no match is found. So I updated that so we can still use this implementation in `queryByText` (which shouldn't throw).